packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -741,7 +783,7 @@
 			"#": "83",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "LED - illuminance for high indicator brightness",
-			"description": "Light intensity level above which brightness of visual indicator is set to 100%.",
+			"description": "When the light intensity (lux) is above this, the brightness of the light is 1%.",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.